Core/Core/Common/CommonUI/CoreWebView/View/CoreWebView.swift

```diff
@@ -350,6 +352,17 @@ open class CoreWebView: WKWebView {
             fileLoadCompletion?(.init(error: error))
         }
     }
+
+    private func registerForTraitChanges() {
+        let traits = [UITraitUserInterfaceStyle.self]
+        registerForTraitChanges(traits) { (webView: CoreWebView, _) in
+            webView.updateInterfaceStyle()
+        }
+    }
+
+    func updateInterfaceStyle() {
+        themeSwitcher?.updateUserInterfaceStyle(with: traitCollection.userInterfaceStyle)
```

Comment:
I'm not really sure what is the difference, but the original code have this distinction:
```suggestion
        let traitCollection = viewController?.traitCollection ?? traitCollection
        themeSwitcher?.updateUserInterfaceStyle(with: traitCollection.userInterfaceStyle)
```